course_dashboard/answers_distribution_reports_manager/utils.py

```diff
@@ -0,0 +1,43 @@
+import os
+import hashlib
+import unicodedata
+
+from django.conf import settings
+
+from fun import shared
+
+def get_answers_distribution_reports_from_course(course_key):
+    """
+    Return a list of reports files ordered by date corresponding to a course
+    """
+    
+    course_path = shared.get_path(settings.ANSWERS_DISTRIBUTION_REPORTS_DIRECTORY, course_key.org, course_key.course)
+
+    if os.path.exists(course_path):
+        files = [f for f in os.listdir(course_path) if os.path.isfile(os.path.join(course_path, f))]
+
+        files.sort(key=lambda s: os.path.getmtime(os.path.join(course_path, s)))
+        return reversed(files)
+    return None
+    
+
+def find_encoded_file_in_dir(encoded_filename, course_key):
+    """
+    Reports names are encoded using has
+    """
+    
+    course_path = shared.get_path(settings.ANSWERS_DISTRIBUTION_REPORTS_DIRECTORY, course_key.org, course_key.course)
+    
+    if os.path.exists(course_path):
+        for f in os.listdir(course_path):
+            if os.path.isfile:
+                current_file_encoded = hashlib.md5(f.encode('utf-8')).hexdigest()
+                if current_file_encoded == encoded_filename:
+                    return f
+        return None
```

Comment:
y'a pas vraiment de règles là dessus... http://stackoverflow.com/questions/15300550/python-return-return-none-and-no-return-at-all